assets/base/i18n/de.json

```diff
@@ -334,6 +334,7 @@
   "mobile.pinned_posts.empty_description": "Wichtige Elemente anheften durch gedrückt halten einer Nachricht und wählen von \"An Kanal anheften\".",
   "mobile.pinned_posts.empty_title": "Keine angehefteten Nachrichten",
   "mobile.post_info.add_reaction": "Reaktion hinzufügen",
+  "mobile.post_info.reply": "Antworten",
```

Comment:
We only need the addition in `en.json` the rest of the languages will be taken care by https://docs.mattermost.com/developer/localization.html#translations-updates

Reply:
also we can just use `post_info.reply` as it already exists and there is no need for yet another key ;)

Reply:
Just to make sure I'm understanding correctly, should I remove all the translations and just keep the `"mobile.post_info.reply": "Reply",` en.json?

Reply:
That is correct